certval/src/validator/path_results.rs

```diff
@@ -27,7 +27,7 @@ pub enum CertificationPathResultsTypes {
     /// Represents validation result
     PathValidationStatus(PathValidationStatus),
     /// Represents error
-    Error(Error),
+    Error(Arc<Error>),
```

Comment:
This is because `Error` isn't clone-able anymore because `der::Error`  doesn't implement clone anymore (https://github.com/RustCrypto/formats/pull/1328)

Reply:
Least to say: I'm not sure this is the correct solution